find_xml_file.rb

```diff
@@ -0,0 +1,15 @@
+
+def find_xml_file(folder)
+  files = Dir.entries(folder)
+
+  xml_file = nil
+
+  files.each do |filename|
+    if filename.match "\.xml$"
```

Comment:
`Dir['*.xml']` offers a convenient way to do this or `Dir['**/*.xml']` in case of subdirectories